spring-data-mongodb/src/main/java/org/springframework/data/mongodb/core/MongoOperations.java

```diff
@@ -56,6 +57,7 @@
  * @author Chuong Ngo
  * @author Christoph Strobl
  * @author Thomas Darimont
+ * @author Mark Paluch
  */
 public interface MongoOperations {
```

Comment:
Please revert changes on `MongoOperations` and `MongoTemplate` completely clients can use the `Criteria`-API to work with examples and use existing methods taking a `Query` instance.